Classes/Finisher/Mail.php

```diff
@@ -480,8 +480,8 @@ protected function parseTemplate(string $mode, string $suffix): string {
     if (isset($this->settings['templateFile'])) {
       $templateCode = $this->utilityFuncs->readTemplateFile('', $this->settings);
     }
-    if (isset($this->settings[$mode]) && is_array($this->settings[$mode]) && isset($this->settings[$mode]['templateFile'])) {
-      $templateCode = $this->utilityFuncs->readTemplateFile('', $this->settings[$mode]);
+    if (isset($this->emailSettings['templateFile'])) {
```

Comment:
please use !empty. 
If no user email configuration was used, the value is set but empty

Reply:
Hi @luzat, Thanks for your contribution.

Could you update your PR to use !empty instead of isset?

Reply:
It seems that it would be preferable for all settings in parseTemplate to fall back to defaults for empty strings, I updated the PR accordingly.